frontend/src/components/photos/styled-components/Mobile.jsx

```diff
@@ -1,9 +1,9 @@
 import styled from 'styled-components';
 
 export default styled.div`
-	display: none;
```

Comment:
a general comment about how you are structuring your component files. For every component, there should ideally be a styles file, a react component, and a test file. Heres an example structure:
- features
  - photos
    -index.js
    -styles.css.js
    -test.js

in this way, your `styles.css.js` file can export all styled components. doing so allows you to describe your react component in detail. heres an example:

export const container = styled.div`/*component container*/`
export const iconContainer = styled.div`/*component icon container*/`
etc...

Reply:
Yes, this is an area that I was experimenting with and trying to find a structure that was consistent and readable. I was taking the approach you mentioned at first, but then I start getting lost in the styles.js files in the same was I get lost in a regular CSS stylesheet. So then I started to think if I have a component that can have its own styled-components, as well as several subcomponents that can have their own styled components, then it makes sense to me to give every React component and every one of its subcomponents their own folder for styled-components to separate the concerns of each component, and try to give each styled-component a descriptive name.